.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 sudo: required
 python:
-  - "2.7"
+  - "3.4"
```

Comment:
Maybe also add a note in the disclaimer that we only test/support python3

Reply:
Done in README.md